go.mod

```diff
@@ -51,6 +51,84 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require (
```

Comment:
Why a separate block? 

Guess those are all indirect. But the above one also still contains indirect ones.

Reply:
Not sure what happened there. Manually cleaned that up and reran `make presubmit`.